opentelemetry-api/src/opentelemetry/metrics/_internal/__init__.py

```diff
@@ -719,6 +719,7 @@ def get_meter(
     name: str,
     version: str = "",
     meter_provider: Optional[MeterProvider] = None,
+    schema_url: Optional[str] = None,
```

Comment:
Can the schema_url addition be moved into a separate PR?

Reply:
Created https://github.com/open-telemetry/opentelemetry-python/issues/3451 to follow up